minidump-processor/src/stackwalker/x86.rs

```diff
@@ -268,24 +268,27 @@ where
             // smaller than 128 KB.
             const MAX_REASONABLE_GAP_BETWEEN_FRAMES: Pointer = 128 * 1024;
 
-            let address_of_bp = address_of_ip - POINTER_WIDTH;
+            let address_of_bp = address_of_ip.checked_sub(POINTER_WIDTH)?;
             let bp = stack_memory.get_memory_at_address(address_of_bp as u64)?;
```

Comment:
Is there a reason these two lines can't be inside the `if i > 0`? This would also eliminate the need for a checked_sub, right?

Reply:
This might be why you had trouble testing your fix on x86...?